### DISCUSSION TITLE
Deploy SvelteKit apps with Bun using a Dockerfile

### DISCUSSION THREAD

**Discussion by maietta** [coollabsio/coolify#1942] [General]
If you are deploying a SvelteKit-built app using Bun, you may have come to realize it's fairly slow.

Why not use a Dockerfile instead?

```
ARG BUN_VERSION=1.1.2

FROM oven/bun:${BUN_VERSION} AS builder

WORKDIR /app

COPY . .

RUN bun i
RUN bun run build
 
FROM oven/bun:${BUN_VERSION}

COPY --from=builder /app/build .

ENV PORT 3000
EXPOSE 3000

USER bun

CMD ["bun", "run", "start"]
```

You may also want to add to a .dockerignore the following:

```
node_modules/
.svelte-kit/
build/
```

Hope this helps someone. If anyone has any improvements that should be made to this, please update.

**Comment by JustMrMendez** [comment]
Did not expect to find you here
